WebdriverIO.md

```diff
@@ -182,10 +185,27 @@ var openTopic = async function(topicName) {
 }
 ```
 
+As you may have noted above that `$$` selector is used by calling a function beacause we need to use `await` prefix with `$$` selector and we cannot place `await $$` directly at top as `await` can only be used inside `async` function and if we will convert the function to `async` we cannot use it as a constructor for defining page objects.
```

Comment:
```suggestion
As you may have noted above that `$$` selector is used by calling a function beacause we need to use `await` prefix with `$$` selector and we cannot place `await $$` directly at top as `await` can only be used inside `async` function and if we convert the function to `async` we cannot use it because of [this](https://stackoverflow.com/questions/51694221/not-a-constructor-with-async-function#:~:text=When%20you%20use%20the%20async,can't%20be%20a%20constructor) error.
```

Reply:
Done.